### DISCUSSION TITLE
Transcoding WavPack to DSF issue

### DISCUSSION THREAD

**Discussion by PhilB-RLB** [gerbera/gerbera#2514] [Q&A] [answered]
Hello there,

I've just installed Gerbera 1.2.0 on Kubuntu 16.04 and I can play from my client (Cabasse Pearl) any flac or dsf files without any problem.

But when I try to transcode WavPack to DSF (as WavPack is able to compress DSF) I've an issue with this (Cabasse Pearl can't natively read WavPack).

Here is the transcoding section of my config.xml file :
```
<transcoding enabled="yes">
    <mimetype-profile-mappings>
      <transcode mimetype="audio/x-wavpack" using="wavpack2dsd"/>
    </mimetype-profile-mappings>
    <profiles>
      <profile name="wavpack2dsd" enabled="yes" type="external">
        <mimetype>audio/x-dsd</mimetype>
        <accept-url>no</accept-url>
        <first-resource>yes</first-resource>
        <accept-ogg-theora>no</accept-ogg-theora>
        <agent command="wvunpack" arguments="%in -o %out"/>
        <buffer size="1048576" chunk-size="131072" fill-size="0"/>
      </profile>
    </profiles>
  </transcoding>
```

Is anybody can provid any help on this issue ?

**Comment by hmoffatt** [comment]
It looks ok to me. How does it fail? Do the logs show anything?

**Reply by PhilB-RLB** [reply]
Thank you for your help. No the logs don't show anything relevant to me.

**Comment by KarlStraussberger** [comment]
1.2.0 is really old and may have issues none of us recollects. You'd better try to get a more recent version which may be complicated because your base distro is old as well and the current version of gerbera depends on rather new libraries and compilers.

**Reply by PhilB-RLB** [reply]
Thank you for your reply.

I will try to install a more recent version on another PC running Kubuntu 20.04 (focal).

**Comment by PhilB-RLB** [comment]
Ok, I've just installed Gerbera 1.10.0 (last version) on KUbuntu 20.04.

Unfortunately I have the same issue.
I've checked the /tmp directory where I can see that transcoding seems to be initiated as I found several grd-tr-xxxxxxxxxxx.dsf file but my Client (Cabasse Pearl) says : "Can't read .... The server is unreachable"

On the other side, if I try to play flac or dsf file with my Client from the Gerbera DLNA server, everything works fine.

What do I have to check in order to debug the transcoding of WavPack to DSF ?

**Reply by KarlStraussberger** [reply]
There is a little manual here: https://docs.gerbera.io/en/latest/transcoding.html#testing-and-troubleshooting

Did you already look there?

**Reply by PhilB-RLB** [reply]
Thank you for the doc, I will first check that.

**Comment by PhilB-RLB** [comment, marked as answer]
Don't worry,

I have found another way with Daphile and an UPnP/DLNA Bridge plugin wich work perfectly.
